app/jobs/custom_delivery_job.rb

```diff
@@ -0,0 +1,14 @@
+class CustomDeliveryJob < ActionMailer::MailDeliveryJob
+  def queue_name
+    mailer, action_name, *other = @arguments
+    if mailer.constantize.critical_email?(action_name)
+      super
+    else
+      custom_queue
+    end
+  end
+
+  def custom_queue
+    ENV.fetch('BULK_EMAIL_QUEUE') { :mailers }
```

Comment:
```suggestion
    ENV.fetch('BULK_EMAIL_QUEUE') { Rails.application.config.action_mailer.deliver_later_queue_name }
```


Il faudrait aussi documenter cette option dans `config/env.optional`